coralproject/talk
# [CORL-478] Porting Danish translations to v5

## What does this PR do?
- Adds Danish translations

## How do I test this PR?

- Change env variable LOCALE to "da"
- See Coral in Danish
